data/json/monsters/zed_misc.json

```diff
@@ -89,6 +89,59 @@
     },
     "flags": [ "SEES", "HEARS", "SMELLS", "KEENNOSE", "STUMBLES", "WARM", "POISON", "GRABS", "NO_BREATHE", "FILTHY" ]
   },
+  {
+    "id": "mon_zombullfrog",
+    "type": "MONSTER",
+    "name": { "str": "zombullfrog" },
+    "description": "The wound covered skin of this large bullfrog appears to be peeling in places, revealing a black goop beneath the surface.  Its vacant stare passes by anything that isn't food, and it will leap to devour anything it can.",
+    "default_faction": "zombie",
+    "species": [ "ZOMBIE", "AMPHIBIAN" ],
+    "volume": "120 L",
+    "weight": "150 kg",
+    "hp": 120,
+    "speed": 50,
```

Comment:
Consider attack_cost of 50, otherwise hitting every other turn will gimp them hard.